lib/Elastica/Query.php

```diff
@@ -356,8 +356,11 @@ public function addScriptField($name, AbstractScript $script)
      */
     public function addAggregation(AbstractAggregation $agg)
     {
-        if (!array_key_exists('aggs', $this->_params)) {
-            $this->_params['aggs'] = array();
+        /**
```

Comment:
I think the 'todo' about PHP versions should be removed from this PR and submitted as a separate PR of changes to change the supported PHP version instead.

The PR can then be merged during the release cycle where the decision is made to upgrade (and the debate about what version can be had in that PR)